### ISSUE TITLE
All dialects: multiple newlines at EOF not truncated

### ISSUE THREAD

**Issue by myschkyna** [sqlfluff/sqlfluff#1621]
## Expected Behaviour
I expect unnecessary newlines at the end of a file (EOF) to be truncated.

## Observed Behaviour
Unnecessary newlines at EOF do not raise any linting statements

Output of parser:
```bash
[L:  1, P:  1]      |file:
[L:  1, P:  1]      |    statement:
[L:  1, P:  1]      |        create_table_statement:
[L:  1, P:  1]      |            keyword:                                          'CREATE'
[L:  1, P:  7]      |            whitespace:                                       ' '
[L:  1, P:  8]      |            keyword:                                          'TABLE'
[L:  1, P: 13]      |            whitespace:                                       ' '
[L:  1, P: 14]      |            table_reference:
[L:  1, P: 14]      |                identifier:                                   'test_table'
[L:  1, P: 24]      |            whitespace:                                       ' '
[L:  1, P: 25]      |            bracketed:
[L:  1, P: 25]      |                start_bracket:                                '('
[L:  1, P: 26]      |                [META] indent:
[L:  1, P: 26]      |                newline:                                      '\n'
[L:  2, P:  1]      |                whitespace:                                   '  '
[L:  2, P:  3]      |                column_definition:
[L:  2, P:  3]      |                    identifier:                               'test_column'
[L:  2, P: 14]      |                    whitespace:                               ' '
[L:  2, P: 15]      |                    data_type:
[L:  2, P: 15]      |                        data_type_identifier:                 'varchar'
[L:  2, P: 22]      |                newline:                                      '\n'
[L:  3, P:  1]      |                [META] dedent:
[L:  3, P:  1]      |                end_bracket:                                  ')'
[L:  3, P:  2]      |    statement_terminator:                                     ';'
[L:  3, P:  3]      |    newline:                                                  '\n'
[L:  4, P:  1]      |    newline:                                                  '\n'
[L:  5, P:  1]      |    newline:                                                  '\n'
```

## Steps to Reproduce
1. Install sqlfluff package version version 0.7.0a4
2. Create file with content below
3. Run sqlfluff lin <path-to-file>

Content of test file test_multiple_newlines.sql:
```sql
CREATE TABLE test_table (
  test_column varchar
);



```

## Dialect
ANSI (but also in other dialects)

## Version
sqlfluff, version 0.7.0a4

## Configuration
```
[sqlfluff]
dialect = ansi
exclude_rules = L009,L011,L029,L031,L034

[sqlfluff:rules]
tab_space_size = 2

[sqlfluff:rules:L010]  # Keywords
capitalisation_policy = upper

[sqlfluff:rules:L014]  # Tables, columns, views
extended_capitalisation_policy = lower

[sqlfluff:rules:L030]  # Function names
capitalisation_policy = upper

[sqlfluff:rules:L040]  # Null & Boolean Literals
capitalisation_policy = upper

[sqlfluff:rules:L016] # Max line length
max_line_length = 120
```

**Comment by barrywhart** [comment]
I think this was addressed in #1838.

**Comment by barrywhart** [comment]
Confirmed that this is handled now. Closing.

**Comment by myschkyna** [comment]
Awesome @barrywhart. Thanks for updating this ticket.
